jupyterlab/labbox_ephys_widgets_jp/src/extensions/timeseries/timeseries.py

```diff
@@ -1,7 +1,12 @@
 import labbox_ephys as le
 
 
-@le.register_recording_view
-def Timeseries(*, sorting: le.LabboxEphysSortingExtractor, recording: le.LabboxEphysRecordingExtractor):
+@le.register_recording_view('Timeseries')
+def Timeseries_recordingview(*, recording: le.LabboxEphysRecordingExtractor):
+    import labbox_ephys_widgets_jp as lew
+    return lew.create_recording_view('TimeseriesView', recording=recording)
+
+@le.register_sorting_view('Timeseries')
```

Comment:
Also register it as a sorting view. In the future we will highlight spike times in the timeseries view.